EtherealEngine/etherealengine
# Added log sending on early refresh/close

## Summary
- [x] `logger.ts` now sends logs if window is refreshed or tab is closed or minimized. ( Not guaranteed to function properly if the whole browser is closed )

## References
#8851 

## QA Steps
✔️ Ran `npm run check-errors`, no related errors found
✔️ Ran `npm run dev`, no errors. 
 


Edit: beforeunload event was changed to visibity event for better compatibility on phones.
